.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+name: release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  goreleaser:
+    name: Release a new version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.16.x"
+
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SOURCE: "file go_bindata github github_ee bitbucket aws_s3 google_cloud_storage godoc_vfs gitlab"
```

Comment:
Is there anyway to get these from the Makefile? We should have a single source of truth for what drivers are bundled.
I *think* we can do this w/ [step](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#steps-context) [outputs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs) e.g. Modify the make file to print it's variables, run that as a new step, use that output in the goreleaser step

Reply:
Yes, I'm absolutely in favor of the single source of truth. I can put that together either before or after we merge this. Going to be later today when I can work on it

Reply:
Let's have a single source of truth before the merging

Reply:
I've been thinking (haven't got the time yet to do work on this) and what if we create a separate file for the "truth" with the source and database drivers and read that in the Makefile into the variables, as well as use it for GoReleaser? Thoughts?

Reply:
Made the changes in the Makefile, added an `echo-source` and `echo-database` step, using those to print drivers and [set them in an environment variable](https://docs.github.com/en/enterprise-server@2.22/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable) which will be available in the next step. Was trying to do it with the step outputs but seemed easier to create the environment variables themselves so the following GoReleaser step will have access to them instead of playing with the outputs.